src/parse:incidents.ts

```diff
@@ -177,56 +180,52 @@ if (!files.length) {
     performance.measure('lsoa', 'lsoa-start', 'lsoa-end');
     performance.mark(`iter-${iter}`);
 
-    console.log(`
-------------------------------------
->>> LSOA (Lower Super Output Areas)
->>> total LSOAs: ${lsoas.size.toLocaleString()}
->>> total POSTCODES: ${postcodes.toLocaleString()}
-`);
+    const out = (final?: boolean) => 
+        output.processingInfo(processedRecords, processedInvalidRecords, incidents.length, queue, final);
 
+    const markersStore: Set<string> = new Set();
     const incidents = [];
+    const markers = [];
     let processingFile = 0;
     
-    function calculateDistance(lat1: number, lon1: number, lat2: number, lon2: number): number {
-        const R = 6371; // Radius of the Earth in kilometers
-        const dLat = (lat2 - lat1) * (Math.PI / 180);
-        const dLon = (lon2 - lon1) * (Math.PI / 180);
-        const a =
-            Math.pow(Math.sin(dLat / 2), 2) +
-            Math.cos(lat1 * (Math.PI / 180)) * Math.cos(lat2 * (Math.PI / 180)) * Math.pow(Math.sin(dLon / 2), 2);
-        const c = 2 * Math.atan2(Math.sqrt(a), Math.sqrt(1 - a));
-        const distance = R * c;
-        return distance;
-    }
+    // function calculateDistance(lat1: number, lon1: number, lat2: number, lon2: number): number {
```

Comment:
I think it can be done in PR, to drop code so you can reverse it later if needed